src/v0/destinations/tiktok_ads_offline_events/data/TikTokTrack.json

```diff
@@ -0,0 +1,43 @@
+[
+  {
+    "destKey": "event_set_id",
+    "sourceKeys": "properties.eventSetId",
+    "required": false
```

Comment:
This is a required field in their documentation. Just curious if there is any reason we have not set it to required.

Reply:
updated this to true. In the doc provided by TikTok, although majority of the fields are mentioned as required, API is throwing error only if `event_set_id`, `event`, `timestamp`.

Reply:
Oh gotcha.